lib/holidays/finder/rules/in_region.rb

```diff
@@ -9,7 +9,15 @@ def call(requested, available)
             # When an underscore is encountered, derive the parent regions
             # symbol and check for both.
             requested = requested.collect do |r|
-              r.to_s =~ /_/ ? [r, r.to_s.gsub(/_[\w]*$/, '').to_sym] : r
+              if r.to_s =~ /_/
+                chunks = r.to_s.split('_')
+                
+                chunks.length.downto(1).map do |num|
```

Comment:
sure, will do as soon as I got time. Right now this is only testet with the definition. Weekend is near, so I hope to bring it in then.